graphql_compiler/__init__.py

```diff
@@ -132,3 +141,36 @@ def graphql_to_gremlin(schema, graphql_query, parameters, type_equivalence_hints
         schema, graphql_query, type_equivalence_hints=type_equivalence_hints)
     return compilation_result._replace(
         query=insert_arguments_into_query(compilation_result, parameters))
+
+
+def get_graphql_schema_from_orientdb_records(schema_records, class_to_field_type_overrides=None,
+                                             hidden_classes=None):
+    """Construct a GraphQL schema from a list of OrientDB schema records.
+
+    Args:
+        schema_records: list of pyorient.otypes.OrientRecord describing the OrientDB schema.
+                        Generated by running ORIENTDB_SCHEMA_RECORDS_QUERY on OrientDB.
+        class_to_field_type_overrides: optional dict, class name -> {field name -> field type},
+            (string -> {string -> GraphQLType}). Used to override the type of a field in the class
+            where it's first defined and all the the class's subclasses.
+        hidden_classes: optional set of Strings, classes to not include in the GraphQL schema.
+
+    Returns:
+        tuple (GraphQL schema object, GraphQL type equivalence hints dict), or (None, None)
+        if there is no schema data in the graph yet.
+        For example, the graph has no schema data if applying schema updates from the very
+        first update. We have to return None because the GraphQL library does not support
```

Comment:
Let's just raise an error instead of `(None, None)`? There are no schema updates as far as the compiler is concerned, that's an implementation detail of the underlying abstractions.